app/models/conference.rb

```diff
@@ -15,4 +17,9 @@ def tickets_left
     confirmed_attendances = attendances.select { |attendance| attendance.confirmed }
     tickets.to_i - confirmed_attendances.size
   end
+
+  def chronology
```

Comment:
This method should be made private; it's just an internal implementation detail / helper that should not be made available as part of the public api.